src/web/app/pages-admin/admin-search-page/admin-search-page.component.html

```diff
@@ -149,18 +149,22 @@
                 </li>
                 <li class="list-group-item list-group-item-info">
                   <strong>Course Join Link</strong>
+                  <button type="button" class="btn btn-primary margin-left-5px btn-sm" (click)="openEmail(student.courseId, student.email, 'STUDENT_COURSE_JOIN')">Send mail</button>
```

Comment:
Not sure if I can link `STUDENT_COURSE_JOIN` to the backend, without hardcoding it here. How do I get the `enum EmailType` to show up in the frontend? Do I add a new class to `src/main/java/teammates/ui/webapi/request`?

Reply:
You can add `src/main/java/teammates/common/util/EmailType.java` to the `classPatterns` of Gradle task `generateApiRequestFormat`

Reply:
Wow thanks!